src/test/java/com/iota/iri/BundleValidatorTest.java

```diff
@@ -79,6 +79,30 @@ public void checkInconsistencyOfConsistentBundle() {
+                "COSNPEACSPESTKTUS9CSMOCWVUBLERQJQGLLA9AYRRXMOVOBULQZZZJPLNELEX9EVZDAFOHBB9FBAYOTDB9YGJAVTDWNPJVVGEPFJ9QBYLXAYZEETY9GMYQRIRAIPWMJUFMFRSQRCEIOWISMBJKGCIGIIYNMXAYLXZIBNKANT9DSQXPBUPWGUJTOMD9JKSNFZJJFBZYBWRWQIZFOZXQWNJTRZTUPHVEGTAOLMBEPXOB9TCEBNIWXQUHNQJZHAOWGVOOOTSCTEUEMXJUODZP9CDLCMEXAW9YJGVNREENETEYXOAKZVRJVUSCRUSRCNZUGUOLCNHQITWYPJVJGPVULASTONDZRDVCNGJSN9DMMUORCOIBTITSPBXSAVTOHSAXHMGH99APRTQIVCEI9NWYKDOZCCWBNZGGCUCISSMGJHJ9OVSFBZVFMBODKRKSLIDESGFDQITCYFARWYTLFKSPLNVQ9LTYFODCHBAJKCWVZNBLOJH9ABFGDFOXHDNHZILINMYMVZYWTFJJOLADPTZRPJEMGPFQNYTFDGOKMDTDSKPTHIDWLMRSSACZMTHUCWVZHCYSPJRATABZ9JHWEBGTTQORGFOYDWUIYFYGOGGTYGYYI9RPKPMCTYNYKHHNWQKJTDQDHTRUYQPMCDDLVUUHUBPWGPMNQYPKHDHGIUSXJGNUXIYGZRGVYPTTV9KXJDLGVVXRIWIGKZJVCQ9DUWAYNDRKYCGKZDEOKZQCXELSQXLGLAPBDIWRTKGSIRNAZ9XFEGMWOWLPUA9TSGYRDOU9GKY9JJDKCJKKGJCRARGSNECJ9SEVAGPPJMBTMGNWSLNSFBOEYMIQBKOVSCTDIWCOQTHBVTZOFNNIYKKJLAROKIRFFPCLSOWO9BVNNJSXEYLJASPUEFKCEI9SGWADNGS9MZAKFTCAYNRVKGDIGYDTPGJXWFAEFGG9BX9XCNBZGQULH9IPPIQ9VFVAXGELUMCTE9TOFOGCTFYBYEBFOKKAESIIHEWLYLQKWDHPYNELJKMZHDSDSK9ZWFP99FQFLUXGTSVDMGHIFZFUCYTYPVEFEDM9TOHCQVIBXARYWZEKTWEOVSBYZTJZRUFZJCLRBTOWPTKY99CYQTGPMAKFYALIWPEQLUB9RGANTKDDMSDSJOTNPFZCXKJUGTJWWPK9FAJGPJOEBRHL9ZU9XRBDJBVMFIZIZFGKETH9RJUSDIOWDZIXFOJLT9DSIAWSCEYBNOGOAZQPZKVZTBXGACCBVQGFNGXNQFTVPUGUOHNBKTUERLSTDXGRKZSATIRHDBSXARYATBHOLUHCQGPRMQGSJUAXUCWKDXAUGORHFVVVDZQEYYGSV9IUM9VTMHJCIKBQD9TQLLYTZHAKDHPFGYMRGJOEDJKQTBBNFCKDEPYTPWUJEOPRJWHAILMVW9AXS9HAT9GCGRXZEHWNDOVGKGPWFXFC9IKLVZJCTGZUJFRHGL9FZWZYP9TPOWGOUTQ9INYBVTD9N9OV9AM9GYRJ99MRWRWLOAKJXMDTFNS9NZQAYGVZRJ9WVSIDRRQLGKH9VPUVKFWZMSRVVLKXUGJAZESC9LEYDNNDWS9WTWNFPCR9YKPOMIJAVHCRCJVBDFWSTDN9BPJRDMGZFXWTY9GUX9WEPJQMXSECFPREXAJ9DU9OLMNQFTF99QYZOGDUTMBHMNMCNUQTU9FTNEUNBT9NQO9YADGY9C9LQR9DYVJZE9GMMYAOUSWUIVJ9EAR9EEFCO9SZTQ9JQHPIJGXIOQZHOSUSVMHBOLTYSEONVLRISNAN9KJHHFFISTYIHYVAMXLPNALDELETXVSWGUWOIVKNAKZKNNRVNJKE9WMNQBXFGXSYZQQSIODAKINA9CPBOGAYKJ9V9GS9HERXSVYPEILYXC9GURBREBCXRZJVXHUOPGYFLPIFBCOTMHFTAAUQUZVEKOWYFFGSCXDFDBTSGZRMPQR9EYSZYGQETCWFDRXVOXXCUBOQTBQGVLAONSTSLURFYSDPAVNCXNWVPQOUORMVSLYISQUQKXFYYMJSGHQP9VAAZABEAGJMCPIEBZQSYCPS9CNKFBDRDTDA9C9MVKGUISRX9ODVUJNPDWMZ9ESYXJ99ETDGSTJZMZYJEGRUWTORMCLNHLDQ9YIJWPMDYGGJQHMFPJ9VKINJUESTCBJQXFTCJFNUXPZONVKXJRAYGKIWHXSBZRKBDLRWXSMKW99KPWJOQYPMHB9KWALSVLFNANYSMNWSJLUOCBKAWAJHNWMLPLXHVEPTKBPWLWHMD999999999999999999999999999999999999999999999999999999FDKW9AD99B99999999B99999999RJPQGE9AXNBBLPQ9IFJXFJTXJAIGDDXAWXGAVU9PJQNYFADJDIDBTJKSAEBBX9NWJGZVAVXYWXPCEFFH9XIFVJWCABR9XQKOWCXJAZDZWMWWBVHROBOPISAWDIJTSFPVAUZNJUMNUCOFIBIAAVIOWDTCNIPRVA9999FTXZAEQIKGFJABOGCCQGOMKNASGWCQWMNIMRDTHKPJMTKDGMEQNZOAYARIYEWFZYJALRKFODOITTZ9999999999999999999999999999999LLHDBWNOF999999999K99999999POWSRVIO9VMIJ99UMGMTTTPNNNV",
+        };
+        List<TransactionViewModel> transactions = persistAndMapTxs(trytes);
+        assertEquals("should be valid because bundle approves tails", BundleValidator.Validity.VALID,  BundleValidator.validateBundleTailApproval(tangle, transactions));
+    }
```

Comment:
2 requests:
1. Can you add some comments explaining the bundles you created?
2. Can you add tests that check for invalidity?

about (1), I like less how the tests in general in this class are written. When Luca refactored the bundle validator I didn't want to delay him and it slipped in. We will not rewrite them to make sure they are clear any time soon, but comments on new tests will be nice :-)